circular_saw.lua

```diff
@@ -13,8 +13,7 @@ circular_saw = {}
 circular_saw.known_stairs = setmetatable({}, {
 	__newindex = function(k, v)
 		local modname = minetest.get_current_modname()
-		print(("WARNING: mod %s tried to add node %s to the circular saw"
-				.. " manually."):format(modname, v))
+		print(S("WARNING: mod @1 tried to add node @2 to the circular saw manually.", modname, v))
```

Comment:
Console print are not translated when printed, this line should not be localized.

Reply:
I had read that you can configure the server in a language other than English

Reply:
it will print in any case: `@moreblocks)` before the localized string.
I only have tested with the client (changing the setting "client and server language") but strings seems to not be translated anyway…

Reply:
Indeed, console messages shouldn't be translated. It's easier to search for error messages online when they're in English anyway :slightly_smiling_face: